lib/main.dart

```diff
@@ -1,11 +1,87 @@
 import 'package:ambience/handlers/file_handler.dart';
 import 'package:ambience/handlers/wallpaper_handler.dart';
 import 'package:flutter/material.dart';
+//install http.dart via 'flutter pub add http'
+import 'package:http/http.dart' as http;
+import 'dart:convert';
+//install via flutter pub add path_provider
+//may need to also enable install loose files on windows developer settings
+import 'package:path_provider/path_provider.dart';
+import 'dart:io';
 
 void main() {
   runApp(const MyApp());
 }
 
+geolocate(var apiKey, String? input) async {
```

Comment:
Same as my comment for 'storage', can we put these functions in their own file(s) and import them? I think it's probably best practice to limit 'main.dart' to include only:
1. the main() function and
2. the class MyApp which handles the UI (starting on line 85)

If we follow this convention of "modulizing" our code I think it will make the codebase easier to maintain

Reply:
Did it, let me know what you think.